data/generate.fsx

```diff
@@ -14,23 +14,28 @@ type Field =
   | Text of string
   | Integer of int64
   | Real of float
+  | Timestamp of DateTime
+
+let quoteString (string: string) =
+  "\"" + string.Replace("\"", "\"\"") + "\""
 
 let fieldToString field =
   match field with
   | Null -> "NULL"
   | Text value -> $"'{value}'"
   | Integer value -> string value
   | Real value -> string value
+  | Timestamp value ->
+    $"""'{value.ToString("yyyy-MM-ddTHH:mm:ss", System.Globalization.CultureInfo.InvariantCulture)}'"""
```

Comment:
Maybe this should also be a function?

Reply:
I decided to offload this work to the `Value.Timestamp` stringification.